gno.land/cmd/gnoland/testdata/panic1.txtar

```diff
@@ -0,0 +1,26 @@
+# test recursion 
+
+## start a new node
+gnoland start
+
+gnokey maketx addpkg -pkgdir $WORK -pkgpath gno.land/r/demo/panic -gas-fee 1000000ugnot -gas-wanted 2000000 -broadcast -chainid=tendermint_test test1
+
+! gnokey maketx call -pkgpath gno.land/r/demo/panic --func Trigger --gas-fee 1000000ugnot --gas-wanted 2000000 --broadcast -chainid=tendermint_test test1
+
+stderr 'panic\(i\<VPBlock\(1\,0\)\>\)'
+stderr 'gno.land/r/demo/panic/panic.gno:5'
+stderr 'p\<VPBlock\(3\,0\)\>\(\)'
+stderr 'gno.land/r/demo/panic/panic.gno:9'
```

Comment:
Please use  `cmp` instead of this stderr comparison. See `gnovm/cmd/gno/testdata/gno_transpile/07_build_flag_with_build_error.txtar` for an example.

Reply:
you can use `UPDATE_SCRIPTS=true` environment variable when you run go test to automatically update cmp golden files output within the txtar file

Reply:
I encountered some elements in the stderr output linked to my desktop path. I don't know of how to address this issue. 

The output:

```
Msg Traces:
    0  /Users/ghost/Documents/projects/teritori/gno/tm2/pkg/crypto/keys/client/maketx.go:213 - transaction delivery failed: log:msg:0,success:false,log:--= Error =--
Data: &errors.errorString{s:"here"}
Msg Traces:
    0  /Users/ghost/Documents/projects/teritori/gno/gno.land/pkg/sdk/vm/keeper.go:318 - VM call panic: here
panic here
panic(i<VPBlock(1,0)>)
    gno.land/r/demo/panic/panic.gno:5
p<VPBlock(3,0)>()
    gno.land/r/demo/panic/panic.gno:9
Trigger()
    gno.land/r/demo/panic/panic.gno:8

Stack Trace:
    0  /Users/ghost/Documents/projects/teritori/gno/tm2/pkg/errors/errors.go:20
    1  /Users/ghost/Documents/projects/teritori/gno/gno.land/pkg/sdk/vm/keeper.go:318
    2  /opt/homebrew/Cellar/go/1.22.3/libexec/src/runtime/panic.go:770
    3  /Users/ghost/Documents/projects/teritori/gno/gnovm/pkg/gnolang/op_call.go:430
    4  /Users/ghost/Documents/projects/teritori/gno/gnovm/pkg/gnolang/machine.go:1173
    5  /Users/ghost/Documents/projects/teritori/gno/gnovm/pkg/gnolang/machine.go:737
    6  /Users/ghost/Documents/projects/teritori/gno/gno.land/pkg/sdk/vm/keeper.go:324
    7  /Users/ghost/Documents/projects/teritori/gno/gno.land/pkg/sdk/vm/handler.go:53
    8  /Users/ghost/Documents/projects/teritori/gno/gno.land/pkg/sdk/vm/handler.go:33
    9  /Users/ghost/Documents/projects/teritori/gno/tm2/pkg/sdk/baseapp.go:650
   10  /Users/ghost/Documents/projects/teritori/gno/tm2/pkg/sdk/baseapp.go:829
   11  /Users/ghost/Documents/projects/teritori/gno/tm2/pkg/sdk/helpers.go:17
   12  /Users/ghost/Documents/projects/teritori/gno/tm2/pkg/sdk/baseapp.go:418
   13  /Users/ghost/Documents/projects/teritori/gno/tm2/pkg/sdk/baseapp.go:391
   14  /Users/ghost/Documents/projects/teritori/gno/tm2/pkg/bft/abci/client/local_client.go:180
   15  /Users/ghost/Documents/projects/teritori/gno/tm2/pkg/bft/appconn/app_conn.go:144
   16  /Users/ghost/Documents/projects/teritori/gno/tm2/pkg/bft/rpc/core/abci.go:59
   17  /opt/homebrew/Cellar/go/1.22.3/libexec/src/reflect/value.go:596
   18  /opt/homebrew/Cellar/go/1.22.3/libexec/src/reflect/value.go:380
   19  /Users/ghost/Documents/projects/teritori/gno/tm2/pkg/bft/rpc/lib/server/handlers.go:185
   20  /Users/ghost/Documents/projects/teritori/gno/tm2/pkg/bft/rpc/lib/server/handlers.go:209
   21  /opt/homebrew/Cellar/go/1.22.3/libexec/src/net/http/server.go:2166
   22  /opt/homebrew/Cellar/go/1.22.3/libexec/src/net/http/server.go:2683
   23  /Users/ghost/go/pkg/mod/github.com/rs/cors@v1.10.1/cors.go:281
   24  /opt/homebrew/Cellar/go/1.22.3/libexec/src/net/http/server.go:2166
   25  /Users/ghost/Documents/projects/teritori/gno/tm2/pkg/bft/rpc/lib/server/http_server.go:216
   26  /Users/ghost/Documents/projects/teritori/gno/tm2/pkg/bft/rpc/lib/server/http_server.go:189
   27  /opt/homebrew/Cellar/go/1.22.3/libexec/src/net/http/server.go:2166
   28  /opt/homebrew/Cellar/go/1.22.3/libexec/src/net/http/server.go:3137
   29  /opt/homebrew/Cellar/go/1.22.3/libexec/src/net/http/server.go:2039
   30  /opt/homebrew/Cellar/go/1.22.3/libexec/src/runtime/asm_arm64.s:1222
--= Error =--
,events:[]
--= Error =--
```